src/components/allUsersSpecific/AllUserBookings.jsx

```diff
@@ -79,8 +82,8 @@ const AllUserBookings = () => {
         apiURL += `?from_loc=${fromValue}&to_loc=${toValue}`;
       }
     } else if (startTime || endTime) {
-      const isoStartTime = startTime?.toISOString();
-      const isoEndTime = endTime?.toISOString();
+      const isoStartTime = startTime ? new Date(startTime).toISOString() : null;
```

Comment:
Is this necessary? 

I think the optional chaining present here should automatically handle it?